crates/bevy_asset/src/handle.rs

```diff
@@ -49,25 +51,28 @@ impl<'a> From<AssetPath<'a>> for HandleId {
 }
 
 impl HandleId {
+    /// Creates a random id for an Asset of type `T`.
     #[inline]
     pub fn random<T: Asset>() -> Self {
         HandleId::Id(T::TYPE_UUID, rand::random())
     }
 
+    /// Creates the default id for an Asset of type `T`.
     #[inline]
     pub fn default<T: Asset>() -> Self {
         HandleId::Id(T::TYPE_UUID, 0)
     }
 
+    /// Creates an arbitrary asset id without an explicit type bound.
     #[inline]
     pub const fn new(type_uuid: Uuid, id: u64) -> Self {
         HandleId::Id(type_uuid, id)
     }
 }
 
-/// A handle into a specific Asset of type `T`
+/// A handle into a specific Asset of type `T`.
 ///
-/// Handles contain a unique id that corresponds to a specific asset in the [Assets](crate::Assets)
+/// Handles contain an unique id that corresponds to a specific asset in the [Assets](crate::Assets)
```

Comment:
Same mistake as before <3

Reply:
Done 👍 And now the only remaining "an unique" in the repo is [here](https://github.com/bevyengine/bevy/blob/main/crates/bevy_input/src/touch.rs#L8).